docs/DeveloperGuide.md

```diff
@@ -769,5 +774,6 @@ testers are expected to do more *exploratory* testing.
       Expected: No guest is edited, because the name is invalid. Error details shown in the status message.
       Status bar remains the same.
 
-   5. Other incorrect edit commands to try: `edit`, `edit x`, `edit 1 rc/hi`, `...`<br>
-      Expected: Similar to previous.
+   5. Other incorrect edit commands to try: `edit`, `edit x`, `edit 1 rc/hi`, `edit ...`<br>
+      Expected: No guest is edited, because the name is invalid. Error details shown in the status message.
```

Comment:
Should this not be solely the name being invalid? As it changed other fields as well

Reply:
yes you are right!